typing/ikinds/axis_lattice.mli

```diff
@@ -0,0 +1,190 @@
+type t
```

Comment:
This module is quite complex - how necessary have you found it to be for performance?